org.eclipse.jdt.ls.tests/projects/singlefile/s1.java

```diff
@@ -0,0 +1,7 @@
+public class s1 {
```

Comment:
Please use Java conventions: [Pascal cased](https://en.wikipedia.org/wiki/PascalCase) files and class names.